_translator-files/po/pt_BR/1-index.po

```diff
@@ -3,17 +3,18 @@
 # This file is distributed under the same license as the PACKAGE package.
 # FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
 #
-#, fuzzy
 msgid ""
 msgstr ""
 "Project-Id-Version: PACKAGE VERSION\n"
-"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
-"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"PO-Revision-Date: 2022-09-04 18:16+0000\n"
+"Last-Translator: gbonaspetti <gbonaspetti@hotmail.com>\n"
```

Comment:
@gbonaspetti Thanks for your translation! I just want to make you aware of the fact that your E-Mail is visible here, currently this can't be changed.